source/uwp/Renderer/lib/XamlBuilder.cpp

```diff
@@ -1491,10 +1491,10 @@ AdaptiveNamespaceStart
         ComPtr<IAdaptiveCardResourceResolvers> resourceResolvers;
         THROW_IF_FAILED(renderContext->get_ResourceResolvers(&resourceResolvers));
 
-        UINT32 explicitWidth = 0, explicitHeight = 0;
-        THROW_IF_FAILED(adaptiveImage->get_Width(&explicitWidth));
+        UINT32 pixelWidth = 0, explicitHeight = 0;
```

Comment:
>explicitHeight [](start = 31, length = 14)

If we're going to call it pixelWidth, let's call it pixelHeight as well

Reply:
thanks for catching this; this shouldn't have changed.

---
In reply to: [186201969](https://github.com/Microsoft/AdaptiveCards/pull/1427#discussion_r186201969) [](ancestors = 186201969)